### DISCUSSION TITLE
new sdk 51.1

### DISCUSSION THREAD

**Discussion by pierrehub2b** [airsdk/Adobe-Runtime-Support#3490] [General]
Hello,

I try to build my desktop app with the new sdk 51.1.
I currently use the sdk 50.2 to build my app without problems.

With the sdk 51.1 for example, the <s:SolidColor ../> in an mxml class is no more found (I can only bound with the mx:SolidColor component).

Is there something to do to fix that with the new 51.1 sdk ?

Thank you for your help

**Comment by 42retfa** [comment]
Hello,

The problem is probably on your side, I just set up a basic project including the spark SolidColor and managed to compiled it with 51.1.1.4. 
 
May be something went wrong during the overlay of the flex SDK with the AIR SDK ?

Vincent.

**Reply by pierrehub2b** [reply]
Hello Vincent,
Thank you for your response and the confirm that it should work, I will overlay the flex SDK again.

Pierre

**Reply by 2jfw** [reply]
Also make sure to use the correct AIR SDK for this, which is the **AIR SDK for Flex Developers - to be overlaid onto a Flex SDK** found on Harman's Download site below the non-Flex AIR SDK

**Reply by pierrehub2b** [reply]
Hello,

Thank you Vincent and Jan, I have followed your suggestions and it works now ...

Thanks a lot
